terraform/nomad-whisper/create-nomad-whisper-configuration.sh

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env bash
+set -x 
+unset SSH_USER
+
+[ -z "$POOL_TYPE" ] && export POOL_TYPE="whisper"
+[ -z "$ROLE" ] && export ROLE="whisper-pool"
```

Comment:
Why not just make the role whisper?  This value will show up in homer, among other places.